src/components/question-genre/question-genre.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import {shape, string, arrayOf, func} from 'prop-types';
+import AnswerGenre from '../answer-genre/answer-genre.jsx';
+
+const QuestionGenreScreen = ({question, onAnswer}) => {
+  const {answers} = question;
+
+  const [userAnswers, setUserAnswers] = useState({});
+
+  const handleGenreAnswer = (id) => {
+    setUserAnswers(() => ({...userAnswers, [id]: !userAnswers[id]}));
```

Comment:
А зачем нам здесь функция?

```suggestion
    setUserAnswers({...userAnswers, [id]: !userAnswers[id]});
```